stdlib/_thread.pyi

```diff
@@ -21,6 +21,24 @@ class LockType:
         self, type: type[BaseException] | None, value: BaseException | None, traceback: TracebackType | None
     ) -> None: ...
 
+    acquire_lock = acquire
```

Comment:
These were missing, for example on 3.8 we can see:

![CleanShot 2024-08-24 at 20 10 47](https://github.com/user-attachments/assets/565c1f20-3619-47aa-8756-329c7b880f5a)

Reply:
They don't seem to be pure aliases at runtime:

```python
_thread.LockType.acquire_lock == _thread.LockType.acquire
>>> False
```

So it's better to just copy the signatures to reflect that fact.

Reply:
Sounds good in that case.